app/utils/types.ts

```diff
@@ -567,6 +568,17 @@ export interface Description {
     description: string;
 }
 
+export interface SerializedTextWithLength {
```

Comment:
This duplicates 
```
export interface SerializedString
```
from `UpdateSerialization`. Please remove the type from `UpdateSerialization` and re-use the type you added here.